src/fiobank/models.py

```diff
@@ -0,0 +1,267 @@
+from __future__ import annotations
+
+import re
+from datetime import date
+from decimal import Decimal
+from typing import Any, Optional, Union, TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from datetime import date as DateType
+else:
+    DateType = date
+
+try:
+    from pydantic import BaseModel, Field, field_validator
```

Comment:
pydantic is a direct dependency of this project now. why do you check if it's available?